### DISCUSSION TITLE
One Slot Camera

### DISCUSSION THREAD

**Discussion by puli2004** [Raytwo/ARCropolis#344] [Ideas] [answered]
Provide details
One Slot camera should allow the camera folders to be unshared as they have slots for them. It would have been the same issue where version 1 of arc was not able to unshar skins correctly. All I know is version 2 of arc have it in a way that kinda gave it that it was able to one slot but it would reshare it to all.
Example:
C05 Has new camera file {
C00, C01 ,C02 C03, C04, C06, C07 : Replace
For version 3 and up I think the problem is that the camera are being unshared but still are getting shared no matter if the camera are in a other slot. Version 2 shows that the camera that gets replaced no matter what slot its on. Version 3 and up only replaces c00 if there a new camera file. In conclusion. All camera files in each slot are the same file.

How should it be implemented?
I can't be sure about it but if I were to guess. The camera files would need to be made in the table if a file was detected in the sd card. Same way how fighters unshare their models and textures.

**Comment by Raytwo** [comment, marked as answer]
For the 3rd time, no.
